### ISSUE TITLE
ReadMore component skip link target should be focusable

### ISSUE THREAD

**Issue by haszari** [woocommerce/woocommerce-gutenberg-products-block#1692]
**Describe the bug**
Storybook a11y addon found an issue which we should look at:

<img width="583" alt="slack-imgs com" src="https://user-images.githubusercontent.com/4167300/73491995-a9fa2880-4414-11ea-915f-a2e8574fb186.png">

**To Reproduce**
Run & view storybook `npm run storybook` `http://localhost:6006/`, view accessibility audit for `ReadMore` component.

**Expected behavior**
No accessibility violations :)

**Comment by dinhtungdu** [comment]
I can't reproduce this issue on my end. The `ReadMore` component doesn't have any Accessibility issue.
<img width="1104" alt="Screen Shot 2021-09-23 at 15 19 02" src="https://user-images.githubusercontent.com/5423135/134475417-388b5e6c-2e46-46c6-a124-1e7336305d2c.png">
